src/ocamlorg/lib/package.ml

```diff
@@ -46,6 +51,46 @@ module Info = struct
       (fun lst (name, cstr) -> (name, string_of_formula cstr) :: lst)
       []
 
+  let depends packages opams f =
+    let open OpamTypes in
+    OpamPackage.Name.Map.fold
+      (fun name vmap acc ->
+        let v =
+          OpamPackage.Version.Map.fold
+            (fun version opam acc ->
+              let env v =
+                match OpamVariable.Full.to_string v with
+                | "name" ->
+                  Some (S (OpamPackage.Name.to_string name))
+                | "version" ->
+                  Some (S (OpamPackage.Version.to_string version))
+                | _ ->
+                  None
+              in
+              let deps =
+                OpamFormula.packages packages
+                @@ OpamFilter.filter_formula ~default:true env (f opam)
+              in
+              OpamPackage.Map.add (OpamPackage.create name version) deps acc)
+            vmap
+            OpamPackage.Map.empty
+        in
+        OpamPackage.Map.union (fun a _ -> a) acc v)
+      opams
+      OpamPackage.Map.empty
+
+  let _get_conflict_set pkgs map =
```

Comment:
Why prepend the functions with `_`? If we dont use them, should we delete them?

Reply:
This is an artefact from the initial work, opam2web supports things like `rev_depopts` which uses all of the same logic as `rev_deps` excepts for the sets of packages brought in see: https://github.com/ocaml-opam/opam2web/blob/master/src/o2wPackage.ml#L449 (we can drop these I don't mind... but might be nice to eventually support it)

Reply:
As we don't use the code for now I'm in favor of removing it -- we can add support for it later